protected/assets/NewAppAsset.php

```diff
@@ -21,6 +21,7 @@ class NewAppAsset extends AssetBundle
     public $depends = [
         IconBundle::class,
         SourceSansProBundle::class,
-        FormBundle::class
+        FormBundle::class,
+        MainBundle::class
```

Comment:
If you include `main.css` via `MainBundle` you should remove / empty these properties from `NewAppAsset`:
- `$basePath`
- `$baseUrl`
- `$css`